src/main/resources/application.yaml

```diff
@@ -77,3 +77,14 @@ identity-account-mapper:
     add-payment-modality: /identityAccountMapper/addPaymentModality
     update-payment-modality: /identityAccountMapper/updatePaymentModality
     account-lookup: /identityAccountMapper/accountLookup
+
+keycloak:
+  contactpoint: http://keycloak.sandbox.fynarfin.io
```

Comment:
These configs to be added to integration test job helm configuration. The url should be replaced with service name